spec/lib/saml_idp/controller_spec.rb

```diff
@@ -91,10 +91,9 @@ def head(status, options = {}); end
     it 'signs a SAML Response if requested' do
       saml_response_encoded = encode_response(principal, signed_response_message: true)
       saml_response_text = Base64.decode64(saml_response_encoded)
-      saml_response = REXML::Document.new(saml_response_text)
-      response_id = REXML::XPath.match(saml_response, '//samlp:Response').first.attributes['ID']
-      signature_ref = REXML::XPath.match(saml_response,
-                                         '//ds:Reference').first.attributes['URI'][1..-1]
+      saml_response = Nokogiri.parse(saml_response_text)
```

Comment:
recommendation: use [`Nokogiri.XML` instead of `Nokogiri.parse`](https://github.com/18F/saml_idp/pull/110/files#r1697541818)

Reply:
thanks! updated in https://github.com/18F/saml_idp/pull/110/commits/512ad9b1609884781034773f6c4ccd53e7036a82